src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -53,10 +51,17 @@ internal int FindMetricAggregators(ReadOnlySpan<KeyValuePair<string, object>> ta
             int len = tags.Length;
             if (len == 0)
```

Comment:
nit: You can probably combine these first two `if`s to reduce a level of nesting eg:

```csharp
if (len == 0 && !this.zeroTagMetricPointInitialized)
```